src/dodal/devices/areadetector/adutils.py

```diff
@@ -0,0 +1,28 @@
+from nslsii.ad33 import CamV33Mixin
```

Comment:
`CamV33Mixin` is very useful for synchronising the AD plugins, but it is not ideal to depend on the `nslsii` package, which brings in a number of dependencies we don't need. It may be worth broaching the idea of moving that functionality into ophyd.

Reply:
I'm hesitant to add anything to `ophyd` as we're about to create all the v2 devices in `ophyd-epics-devices`...

Reply:
Reasonable, it's not a big class, I wonder how they'd feel about us duplicating it in Dodal

Reply:
I think it makes sense to copy it for now and then refactor to use a shared version into `ophyd-epics-devices` when we are ready. The goal with `nslsii` is to hold devices (and other things) that are not useful outside NSLS2 or that are not yet at the standard (tests, etc.) of ophyd, so in general our intent is to hoist things like this upstream when they are broadly useful, rather than introduce a dependency on `nslsii`.